src/components/unsigned-sweep.js

```diff
@@ -176,15 +180,17 @@ class UnsignedSweep extends Component {
       };
 
       // Retrieve the desired file path and file name
-      const filePath = dialog.showSaveDialog(dialogParams);
+      const filePath = await dialog.showSaveDialog(dialogParams);
       if (!filePath) {
         // TODO: The user exited the file creation process. What do we do?
         return;
       }
 
-      fs.writeFileSync(filePath, JSON.stringify(recoveryPrebuild, null, 4), 'utf8');
-      this.setState({ recovering: false, done: true, finalFilename: filePath });
+      fs.writeFileSync(filePath.filePath, JSON.stringify(recoveryPrebuild, null, 4), 'utf8');
+      this.setState({ recovering: false, done: true, finalFilename: filePath.filePath });
+      alert('We recommend that you use a third-party API to decode your txHex and verify its accuracy before broadcasting.');
     } catch (e) {
+      console.dir(e);
```

Comment:
Leftover or is this how we do logging in wrw? Just double checking.

Reply:
This logs the error to the console if there are any errors (which we would ask the customers to take a screenshot for)...There really isn't a standard practice as for how we log errors. Happy to hear if you have any other ideas on how we can do this!

Reply:
I would just wrap it in a `function log(s) { console.log(s) }` or something. Than it's self-explanatory for the reader that it's intentional. The function can be in some `src/log.js` file.

Reply:
Good idea, fixed!